content/en/news/2024/stream-recap-8-sep.org

```diff
@@ -8,14 +8,14 @@
 #+SUMMARY: Tried another day with two streams. Worked out a bit better! Well, probably because I had a holiday, but still. Really fun week that even included finishing Pokemon Sword!
 
 * Art Stream
-Started the week with my last kitten-sona (for now). This was of a favorite special viewer of mine who had a very unique idea for their kitten. I love it and I think they do to! It was really fun to draw!
+Started the week with my last kitten-sona (for now). This was of a special viewer of mine who had a very unique idea for their kitten. I love it and I think they do to! It was really fun to draw!
```

Comment:
that make since, good catch! though he could be favorite _and_ special, probably best to pick one 😝